packages/lit-auth-client/src/lib/relay.ts

```diff
@@ -1,16 +1,36 @@
+import {
+  LitNetwork,
+  RELAY_URL_CAYENNE,
+  RELAY_URL_HABANERO,
+  RELAY_URL_MANZANO,
+  RELAY_URL_DATIL_DEV,
+} from '@lit-protocol/constants';
 import {
   IRelay,
   IRelayFetchResponse,
   IRelayMintResponse,
   IRelayPollStatusResponse,
   LitRelayConfig,
 } from '@lit-protocol/types';
+
 import { log } from './utils';
 
 /**
  * Class that communicates with Lit relay server
  */
 export class LitRelay implements IRelay {
+  /** URL for Lit's relay server */
+  static getRelayUrl(litNetwork: LitNetwork): string {
+    const networkMap: Record<LitNetwork, string | undefined> = {
+      [LitNetwork.Cayenne]: RELAY_URL_CAYENNE,
+      [LitNetwork.Manzano]: RELAY_URL_MANZANO,
+      [LitNetwork.Habanero]: RELAY_URL_HABANERO,
+      [LitNetwork.DatilDev]: RELAY_URL_DATIL_DEV,
+      [LitNetwork.Custom]: undefined,
```

Comment:
Good point. I've added a `throw` for the case where `relayUrl` is not defined instead of returning the cayenne value as that could hide runtime typos or other errors

I'll keep `LitNetwork.Custom` so we can keep the `networkMap: Record<LitNetwork, string | undefined>` and have it require every `LitNetwork` without complex types. It does not change the behaviour anyway

Reply:
Let's merge `staging/v7` in  (I just merged master into staging/v7) and use this mapper as we are deprecating the olds:
https://github.com/LIT-Protocol/js-sdk/blob/1a843e951ff5cb1d9203075fe829434bd7f471d4/packages/constants/src/lib/constants/constants.ts#L730-L740

Reply:
Done